### PR TITLE
v0.0.2 Improved

### PR DESCRIPTION
**Pull request by Pitometsu**
- Add support for custom title view
- Fix proper behavior for left and right bar
  button items.

### CODE REVIEW

**Comment by Pitometsu** [comment]
First of all, thanks for awesome control. It's very helpful.
Could you merge it to master, add 0.0.2 tag and make `pod trunk push` for a new version, please?

I work on https://github.com/Pitometsu/CruiserWebViewController
and have add some general compatibility to your code.

Best regards.

File: Classes/IPNavBarSqueezableViewController.m [hunk 1]
```diff
@@ -8,36 +8,51 @@
...
+static const CGFloat kSqueezedNavigationBarHeight = 20.f;
+static const CGFloat kStatusBarHeight             = 20.f;
+static const NSTimeInterval kAnimationDuration    = 0.25;
 
-@interface IPNavBarSqueezableViewController () <UIScrollViewDelegate>
 
 typedef NS_ENUM(NSInteger, IPNavBarSqueezingStatus) {
     IPNavBarSqueezingStatusNormal,
+    IPNavBarSqueezingStatusProgress,
     IPNavBarSqueezingStatusSqueezing,
```
Comment (zetachang) [inline Classes/IPNavBarSqueezableViewController.m:35]: Turns out `IPNavBarSqueezingStatusSqueezing` has been replaced by `IPNavBarSqueezingStatusProgress`. Maybe we should removed this status?

**Comment by zetachang** [comment]
Hi, thank you for taking time working on this :+1: 

I've run the example project. And it turns out the squeezed title label is a little bit higher than the original one.

I've pushed some fix to https://github.com/zetachang/IPSqueezableViewController/commit/690af53384f9d96533574bfef5a48dd93497dcf6. It will fix it, but I am not sure if it's still compatible with the `addressField` one you integrate with in your repo.

Hope you can check it out. If it works, we can merge this and release a new version. :wink: 

Regards.

File: Classes/IPNavBarSqueezableViewController.m (hunk 1 above)
Comment (Pitometsu) [inline Classes/IPNavBarSqueezableViewController.m:35]: As I understand, it both required: `IPNavBarSqueezingStatusProgress` marks manual slow scrolling at:

``` objc
- (void)squeezeNavBarWithProgress:(CGFloat)delta
```

And `IPNavBarSqueezingStatusSqueezing` still mark complete animation status.

**Comment by Pitometsu** [comment]
@zetachang It's ok with custom tittleView, thank you for fast feedback!

**Comment by zetachang** [comment]
`v0.0.2` has been published. Thank you again for your contributions!

**Comment by Pitometsu** [comment]
Thank you for attention!

**Pull request merged**